### ISSUE TITLE
Opening a project from Github Repo shows a "Git Failed with fata Error"

### ISSUE THREAD

**Issue by ReportDataCode** [MicrosoftDocs/visualstudio-docs#6261]
[Enter feedback here]
Here is the error dialog box's content.
Git failed with fatal error.
'
![image](https://user-images.githubusercontent.com/69062810/103454669-47469680-4d21-11eb-80fe-b6e6ab8d4365.png)

I repaired my visual studio after this happen still not fix

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 01e6eab2-1a88-ae5d-9f79-01580ce5eab3
* Version Independent ID: 2fff5602-5e7e-71c9-4a75-0a7339528ecb
* Content: [Tutorial: Open a project from a repo - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/get-started/tutorial-open-project-from-repo?view=vs-2019#feedback)
* Content Source: [docs/get-started/tutorial-open-project-from-repo.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/get-started/tutorial-open-project-from-repo.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

**Comment by ReportDataCode** [comment]
Within my knowledge,the problem seems to be about SSL and certifications.This is a odd for me at least consideration that Cloning a repo is state as "Getting Started".
For my reasons,Getting started should note the SSL and certification for people such as me,a beginner will be informed,

**Comment by TerryGLee** [comment]
@ReportDataCode Thanks for your feedback; I'm working on updating this page for the new [Git experience in Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/git-with-visual-studio?view=vs-2019) and will add the [GitHub sign-in](https://docs.microsoft.com/en-us/visualstudio/ide/work-with-github-accounts?view=vs-2019) info to it, too. I'll report back and close this issue after the updates to the tutorial are published.

Otherwise, for further support for error messages, please note that you can report product issues via the [Visual Studio Developer Community](https://developercommunity.visualstudio.com). 

And, here are some other resources for general info:
- [Visual Studio on Q&A](https://docs.microsoft.com/en-us/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

**Comment by TerryGLee** [comment]
@ReportDataCode We've updated the tutorial to add specific information about how to use the new Git experience to do open a project from a repo. That information includes a sign-in step. But we realized we could do a little bit more. Today, we've added links to additional sign-in info, as follows:

"For more information about signing in to Visual Studio, see the [**Sign in to Visual Studio**](https://docs.microsoft.com/visualstudio/ide/signing-in-to-visual-studio?view=vs-2019) page. And for specific information about how to use your GitHub account to sign in, see the [**Work with GitHub accounts in Visual Studio**](https://docs.microsoft.com/en-us/visualstudio/ide/work-with-github-accounts?view=vs-2019) page."

That new info should be published within the next 24 hours. Meanwhile, we'll close this issue now, with our thanks for helping us improve the documentation!
